grpc/diatheke.proto

```diff
@@ -263,4 +302,39 @@ message TTSAudio {
     bytes audio = 1;
 }
 
+// Data to send to the Transcribe stream. The first message on
+// the stream must be the model ID, followed by audio data.
+message TranscribeInput {
+    oneof data {
+        // ASR model to use for transcription.
+        string cubic_model = 1;
+
+        // Audio data to transcribe.
+        bytes audio = 2;
+    }
+}
+
+// The result from the Transcribe stream. Usually, many partial
+// (or intermediate) transcriptions will be sent until the final
+// transcription is ready for every utterance processed.
+message TranscribeResult {
+    // The transcription.
```

Comment:
Is this formatted text?

Reply:
Could be. Depends on whether the underlying cubic model is using a formatter.